lib/Viroverse/ISLAWorksheet.pm

```diff
@@ -253,7 +267,6 @@ sub make_xlsx {
 
     $xls->close() or die "Error saving Excel file: $@";
 
-    $buffer->setpos(0);
```

Comment:
My intuition is that you should leave the rewind in this file, so that the caller just gets an open filehandle ready to be read.

On the other hand, `IO::String` lets you pass in a buffer initialized earlier rather than letting it initialize an internal one, so if we do something like

```perl
my $buf;
my $io = IO::String->new($buf);
# write stuff
return $buf;
```

the caller only has to deal with a string, not a filehandle, which seems simpler in a few ways. Up to you if you want to try this.

Reply:
Yeah that does seem like an improvement. I wasn't sure if the binding to `$buf` would stick because Perl strings behave differently than strings in pretty much every other language.